js/views/pauseScreen.js

```diff
@@ -54,6 +61,18 @@ var PauseScreen = React.createClass({
     this.props.handleVrPlayerClick();
   },
 
+  /**
+   * remove the button on pause screen for correct checking mouse movement
+   */
+  hideVrPauseButton: function() {
+    if (this.props.controller.videoVr) {
+      var pauseButton = document.getElementById('oo-pause-button');
+      setTimeout(function() {
+        pauseButton.style.display="none";
```

Comment:
Can we add a null check for pauseButton?

Reply:
pauseButton is always exists in the component, but we can add the checking.